controllers/front/OrderController.php

```diff
@@ -356,4 +359,30 @@ public function displayAjaxAddressForm()
             ),
         ]));
     }
+
+    /**
+     * Return default TOS link for checkout footer
+     *
+     * @return string|bool
+     */
+    protected function getDefaultTermsAndConditions()
+    {
+        $cms = new CMS((int) Configuration::get('PS_CONDITIONS_CMS_ID'), $this->context->language->id);
+
+        if (!$cms->id || !Validate::isLoadedObject($cms)) {
```

Comment:
```suggestion
        if (!Validate::isLoadedObject($cms)) {
```

The first check is already done in `isLoadedObject` :sweat_smile:

Reply:
Exat! I had only added the first one at the beginning and added the `isLoadedObject` then, I forgot to remove it! Thank you! :)